### DISCUSSION TITLE
Search & Sort by number of commits

### DISCUSSION THREAD

**Discussion by wakaztahir** [community/community#63611] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

https://github.com/isaacs/github/issues/1206

**Comment by wakaztahir** [comment]
Most repositories on Github have a single commit, people started and never worked on projects, Every time you search for a good project, all these other projects come up.

You can search by stars unless you are searching for a repository that is expected to have less stars.

So what I want is search & sort by number of commits, which can be done along with number of stars to show repositories that are hidden gems !

**Comment by mdnurulmomen** [comment]
Order by commits might be another feature could be added. 
But, number of commits doesn't always shows the authenticity of a repository. Someone may have less important repositories with maximum commits. 
For now, Most forks, Most Stars might be alternatives.

**Reply by wakaztahir** [reply]
Its still useful in skipping insanely large number of repositories that have less commits

**Reply by mdnurulmomen** [reply]
You're right. 
In that case, You may open a discussion related GitHub feature request.

**Reply by wakaztahir** [reply]
Where do I post that ?

**Reply by mdnurulmomen** [reply]
Right here, staring a new discussion, [e.g](https://github.com/orgs/community/discussions/28666). 

You may find the right place from [here](https://github.com/orgs/community/discussions).

May [navigate](https://support.github.com/request) to know more.

**Comment by ShahriarAhmod** [comment]
better you search ...

**Reply by wakaztahir** [reply]
Don't understand what you mean by ...

I know that ... Range search exists but it's not available for commit numbers.
